apprentice-react-o-shop/src/styles/styles.scss

```diff
@@ -1,5 +1,9 @@
 @import 'bootstrap.min.css';
 
-body {
-    display: inline-block;
+#LoginForm {
```

Comment:
Lets find a way to rewrite lines 3-8. Two forms with the same css properties? Hmmm.... looks a little DRY to me